### DISCUSSION TITLE
Unknown column 'id' in 'field list'

### DISCUSSION THREAD

**Discussion by mftaa** [sequelize/sequelize#16117] [Help & Questions] [answered]
hi, i just make a model user with sequelize in mysql database this my code 
`import { Sequelize } from "sequelize";
import db from "../config/Database.js";

const { DataTypes } = Sequelize;

const User = db.define(
  "user",
  {
    KodeID: DataTypes.STRING,
    Nama: DataTypes.STRING,
    userid: DataTypes.STRING,
    password: DataTypes.STRING,
    Level: DataTypes.STRING,
  },
  {
    freezeTableName: true,

  }
);

export default User;`

but i got error when i try triggered the api 
Executing (default): SELECT `id`, `KodeID`, `Nama`, `userid`, `password`, `Level`, `createdAt`, `updatedAt` FROM `user` AS `user`;
Unknown column 'id' in 'field list'

i just confused in this error cause in model there is not id, but in executing there are error in id, what happend in here?

**Comment by ephys** [comment, marked as answer]
Sequelize automatically adds a primary key if your model does not define one: https://sequelize.org/docs/v6/other-topics/legacy/#primary-keys
